applications/DeepSpeed-Chat/training/utils/ds_utils.py

```diff
@@ -24,6 +25,11 @@ def get_train_ds_config(offload,
                         tb_name=""):
 
     device = "cpu" if offload else "none"
+    dtype_extra = {
+       "enabled": True,
+       "loss_scale_window": 100
```

Comment:
`loss_scale_window` only applies to fp16.

Reply:
Update to fix this issue, thanks a lot.